### ISSUE TITLE
[FEATURE REQUEST] Option to process each line as a separate prompt if "Submit on enter?" is disabled

### ISSUE THREAD

**Issue by athu16** [hlky/stable-diffusion-webui#187]
**Is your feature request related to a problem? Please describe.**
Currently the only way to process a lot of prompts sequentially is to use the command line interface and load them from a file. It would be helpful if we could do it through the GUI as well.

**Describe the solution you'd like**
An option which splits the contents of the "Prompt" text box at newlines and treats them as individual prompts, so the user can write all prompts they desire, pressing ENTER after each one, and have the script generate all of them individually.

**Describe alternatives you've considered**
Using a special character to separate prompts (like | is used to split during prompt matrix generation), where prompts will be written one after another.
eg. PROMPT_1 ## PROMPT_2 ## PROMPT_3

**Additional context**
None

**Comment by hlky** [comment]
![image](https://user-images.githubusercontent.com/106811348/187074520-9ba33804-5936-4422-93e5-63d29f6848a1.png)

I think this should allow you to press enter in the prompt box and enter one prompt per line

**Comment by athu16** [comment]
That's true, but currently it processes the entire string as a single prompt
![memmon](https://user-images.githubusercontent.com/93036639/187074896-1eb55314-06d1-447c-9546-33d93d272395.png)

**Comment by hlky** [comment]
Ok thanks for clarifying, it's just not implemented for some reason.

**Comment by JustOxlamon** [comment]
For now you can use new prompt syntax
check `create prompt matrix...` option
and
prompt: @(a truck overflowing with|an aeroplane made out of|big fat rat eating) (cheese|cookie)

this will produce:
a truck overflowing with cheese
an aeroplane made out of cheese
big fat rat eating cheese
a truck overflowing with cookie
an aeroplane made out of cookie
big fat rat eating cookie
